launch_testing_ros/test/examples/set_param_launch_test.py

```diff
@@ -69,7 +69,7 @@ def set_parameter(self, state=True, timeout=5.0):
         request = SetParameters.Request()
         request.parameters = parameters
         future = client.call_async(request)
-        rclpy.spin_until_future_complete(self, future)
+        rclpy.spin_until_future_complete(self, future, timeout_sec=5.0)
```

Comment:
There's already a `timeout` being passed to `set_parameter`, which seems to be totally unused right now.  Should we use that instead?

Reply:
Use timeout parameter in 887ffac1d101056d03db5220e29fe324e89e8fd8